ledger/accountdb.go

```diff
@@ -4826,6 +4681,156 @@ func loadTxTail(ctx context.Context, tx *sql.Tx, dbRound basics.Round) (roundDat
 	return roundData, roundHash, expectedRound + 1, nil
 }
 
+func deleteStoredCatchpoint(ctx context.Context, e db.Executable, round basics.Round) error {
+	f := func() error {
+		query := "DELETE FROM storedcatchpoints WHERE round=?"
+		_, err := e.ExecContext(ctx, query, round)
+		return err
+	}
+	return db.Retry(f)
+}
+
+func storeCatchpoint(ctx context.Context, e db.Executable, round basics.Round, fileName string, catchpoint string, fileSize int64) error {
+	f := func() error {
```

Comment:
why moving these functions and changing the inlined Retry's callback to a named function? Readability? Questionable. But a bigger diff for no reason